src/components/pickers/types.tsx

```diff
@@ -18,7 +18,7 @@ export type DriverPickerProp = {
   watch?: any;
   isShipment?: boolean;
   setValue: (name: string, value: unknown, config?: Object) => void;
-  mode?: string; // possible values: 'Edit', 'Add', ''
+  mode?: 'Edit' | 'Add' | '';
```

Comment:
Just to know - this is just value so better write it from small letter :)

Reply:
I think that those 'mode' values in another places started from big letter and I would rather keep such convention. This is only string value not the variable name.